source/MRMesh/MRPolylineTopology.h

```diff
@@ -14,6 +14,20 @@ namespace MR
 class PolylineTopology
 {
 public:
+    /// data of every half-edge
+    struct HalfEdgeRecord
```

Comment:
Which function requires making this public?

Reply:
`edges()`